### DISCUSSION TITLE
Integrazione in HASSIO di smartplug T31 di EZVIZ

### DISCUSSION THREAD

**Discussion by adellazassa** [indomus/forum#1013] [Hardware & Firmware]
Premesso che sto muovendo i primi passi nella domotica, in HA, e anche in github, chiedo se possibile un aiuto perchè ho comprato, un po' a caso, una T31 di EZVIZ ma ho scoperto che non riesco ad integrarla in Homeassistant pur avendo una versione aggiornata
![immagine](https://user-images.githubusercontent.com/45533529/140641527-3a383e76-a118-419b-8709-a94412d31c17.png)

dopo aver cercato e scelto e compilato l'integrazione ezviz di HA, all'invio mi restituisce un messaggio che non spiega nulla del problema riscontrato
![immagine](https://user-images.githubusercontent.com/45533529/140641718-b562c956-097d-4a2d-b7f2-07591a916f1d.png)
Eppure utenza e pwd utilizzae sono corrette. Sono quelle usate con la app di ezviz che riesce a controllare correttamente la smart plug.
Sbaglio qualche cosa io oppure ho fatto un acquisto azzardato in ottica HA?
Grazie in anticipo per la risposta. Qualunque essa sia.

**Comment by Moxser** [comment]
Ciao. Hai per caso attivo la verifica a due fattori dall'app di ezviz?

**Reply by adellazassa** [reply]
Si.. deve essere anche una impostazione di default. Non ricordavo di averla attivata.
![immagine](https://user-images.githubusercontent.com/45533529/140700650-df1e2e57-69d6-4e10-8f80-f4ae44ed6296.png)

**Reply by adellazassa** [reply]
Comunque ho provato a toglierla e la registrazione è andata a buon fine. Grazie 10^3 Moxser
Ma dopo la registrazione posso riattivare l'autenticazione a 2 fattori?

**Reply by Moxser** [reply]
Io poco tempo fa ho provato a riattivarla ma non funzionava e ho dovuto toglierla nuovamente.
